opa-audit-operator/metadata.yaml

```
@@ -3,9 +3,20 @@ description: |
   This charm deploys the Gatekeeper Audit.
 summary: |
   This charm deploys Gatekeeper Audit
-series: [kubernetes]
+containers:
+  gatekeeper:
+    resource: gatekeeper-image
+    mounts:
+    - storage: tmp-volume
```

```suggestion
    - storage: audit-volume
```
It's probably better to give the volume a more descriptive name like audit volume so that its clear what the volume is intended for